keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -63,67 +63,64 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 
 ## Summary
 
-We will add support in kubelet for the `pullIfNotPresent` image pull policy, for
-ensuring images pulled with pod imagePullSecrets are re-authenticated for other
-pods that do not have the same imagePullSecret/auths used to successfully pull
-the images in the first place.
+We will add support in the kubelet for an admin to enable the ability to ensure an image that is already present on a node because
+a pod with `ImagePullSecrets` previously pulled it is reauthenticated when a new pod with different `ImagePullSecrets` attempts to use the same image,
+when the `ImagePullPolicy` is `IfNotPresent`.
 
-This policy change will have no affect on the `pull always` image pull
-policy or for images that are preloaded.
+In other words: ensure the pull secrets are rechecked for each new set of credentials, and ensure a pod has access to those images.
 
-However, for the `pull never` policy if a first pod successfully pulled an image
-with credential and then a second pod with pull never tried to use the image,
-when the feature gate is on the second pod will receive an error message, where
-before and with the feature gate off the second pod would be able to use the image
-pulled with credentials by the first pod.
+For the `Never` policy, the behavior also must change. Otherwise, a user who wishes to use the image of another pod could just use `Never` and hope
```

Comment:
```suggestion
For the `Never` policy, the behavior also must change, when opted in. Otherwise, a user who wishes to use the image of another pod could just use `Never` and hope
```